contents.json

```diff
@@ -17175,6 +17175,31 @@
       ],
       "date_added": "Dec 6 2018",
       "suggested_by": "@HeiKki"
+     },
+     {
+      "title": "ArtShredder",
+      "category-ids": [
+        "photo",
+        "content"
+      ],
+      "description": "Banksy Shredder for iOS",
```

Comment:
@marty-suzuki could you remove "for iOS" from the description because there are only apps for iOS, it's meaningless.